controller/features_test.go

```diff
@@ -497,14 +503,40 @@ func TestListFeatures(t *testing.T) {
 			// when
 			ctx, err := createValidContext("../test/private_key.pem", "user_beta_level", time.Now().Add(1*time.Hour))
 			require.NoError(t, err)
-			_, featuresList := test.ListFeaturesOK(t, ctx, svc, ctrl, nil, []string{"FeatureX", "FeatureY", "FeatureZ"}, nil)
+			_, featuresList := test.ListFeaturesOK(t, ctx, svc, ctrl, nil, []string{"FeatureX", "FeatureY", "FeatureZ"}, nil, nil)
 			// then
 			expectedData := []*app.UserFeature{}
 			assert.Equal(t, expectedData, featuresList.Data)
 		})
 
 	})
 
+	t.Run("list by strategy", func(t *testing.T) {
+
+		t.Run("1 match", func(t *testing.T) {
```

Comment:
could you add a test in which there's no match ? I assume the response would be `200 OK` with an empty list ?